src/t2iapi/metric/service.proto

```diff
@@ -68,8 +68,10 @@ service MetricService {
       returns (BasicResponse);
 
   /*
-  Set metric values within a given range. If no limit value is defined, random but valid metric values shall be set.
-  If a limit is defined, it shall not be violated by metric values.
+  Set new metric values within a given range. The range is defined by lower and upper limit values which shall not be
```

Comment:
Should we mention that pm:AbstractMetricValue/@Value or @Samples are meant to be set? So it can be only one metric value or more than one value. Or it is enough to write metric value(s)?

Reply:
yes, that makes sence, i adapted the description

Reply:
makes sence, adapted wording